.github/workflows/artifacts.yml

```diff
@@ -1,14 +1,6 @@
 name: artifacts
 
-on:
```

Comment:
Assuming this change should be reverted since the intention is for this to only run on Android changes to master?

Reply:
Yes, just testing here